homeassistant/components/zwave.py

```diff
@@ -171,6 +205,15 @@ def value_added(node, value):
             # Ensure component is loaded
             bootstrap.setup_component(hass, component, config)
 
+            # Configure node
+            name = "{}.{}".format(component, _object_id(value))
+
+            customize = config[DOMAIN].get(CONF_CUSTOMIZE, {})
```

Comment:
You could push this to the root of the `setup` method as it is not supposed to change anymore.